### PR TITLE
feat: Connect historical month to server and display

### LINKED ISSUE #48: feat: Connect historical/month to server and display

### PR DESCRIPTION
**Pull request by jonlld**

### CODE REVIEW

File: src/components/organisms/History.tsx [hunk 1]
```diff
@@ -62,32 +62,34 @@ export default function History() {
 
   const device_id = "a7382f5c-3326-4cf8-b717-549affe1c2eb";
 
+  const updateDataState = (data: []) => {
```
Comment (Takashicc) [inline src/components/organisms/History.tsx]: Make the `data` type clear.
I'm unable to know what's in the data.
Reference the fastapi docs to know what kind of structure will be returned.

File: src/components/organisms/History.tsx [hunk 2]
```diff
@@ -62,32 +62,34 @@ export default function History() {
...
+    const tempMax: number[] = [];
+    const tempMin: number[] = [];
+    const humidMax: number[] = [];
+    const humidMin: number[] = [];
+    const label: string[] = [];
+
+    data.forEach((day: any) => {
```
Comment (Takashicc) [inline src/components/organisms/History.tsx]: Don't use `any`.
Make the type clear.
Reference the fastapi docs to know what kind of structure will be returned.

**Review by Takashicc — CHANGES_REQUESTED** [review]

File: src/components/organisms/History.tsx (hunk 1 above)
Comment (jonlld) [reply src/components/organisms/History.tsx]: Done

File: src/components/organisms/History.tsx (hunk 2 above)
Comment (jonlld) [reply src/components/organisms/History.tsx]: Done

**Review by Takashicc — APPROVED** [review]

**Pull request merged**